mkosi/resources/.gitignore

```diff
@@ -0,0 +1 @@
+/mkosi.1
```

Comment:
Let's just add this to the global gitignore? We don't want this in the repo regardless of the directory its in